database/USAFacts_collector

```diff
@@ -0,0 +1,105 @@
+############################################################################################################################
+################################## Pulling and Transforming USAFacts County Level Data ##################################### 
+############################################################################################################################
+# Script for taking USAFacts county-level dataset, adding lat/long coordinates for each county, and transforming datasheet
+# into NECSI time-series format
+# Only one required package:
+library(readr)
+
+#### Set working directory ####
+setwd("~/Documents/Data_and_Projects/NECSI-COVID19/Data/USA_CountyLevel")
+
+#### Load data ####
+### USAFacts
+cases_url <- "https://static.usafacts.org/public/data/covid-19/covid_confirmed_usafacts.csv?_ga=2.104843072.1960452769.1585151006-660813358.1585151006"
+covid_usa_cases <- read_csv(url(cases_url))
+
+deaths_url <- "https://static.usafacts.org/public/data/covid-19/covid_deaths_usafacts.csv?_ga=2.62244076.1960452769.1585151006-660813358.1585151006"
```

Comment:
what is the ga parameter?

Reply:
Not sure, that was included in the html path to both csv files when I copied them from the USAFacts website?

Reply:
Never mind, looks like it might have been something wonky with the website at the time. Current html paths don't have that ga parameter

Reply:
OK, glad that was cleared up